CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased][latest]
 ### Added
-- Nodejs: Include source snippet in backtraces when available (#624) 
+- Nodejs: Include source snippet in backtraces when available (#624)
```

Comment:
I removed the changelog line in https://github.com/honeybadger-io/honeybadger-js/pull/665/commits/1a35edb9fd0a5f33b77911a1b8e0729c1d98f413 — we can skip the changleog for this PR since it's not actually changing the lib.